### DISCUSSION TITLE
How do I use a rouge theme?

### DISCUSSION THREAD

**Discussion by brandoncc** [bridgetownrb/bridgetown#476] [Q&A] [answered]
I have a basic bridgetown app that I'm working on, no styling at all yet. The first colors I'm trying to get working are syntax highlighted blocks. I have the html output from rouge, but there are no css styles for the tags it adds. How do I get a rouge theme working?

Thanks!

**Comment by jaredcwhite** [comment, marked as answer]
@brandoncc You can download and import one of the CSS files here:
https://jwarby.github.io/jekyll-pygments-themes/languages/ruby.html

**Reply by brandoncc** [reply]
Thank you, I must have missed it somewhere in the documentation.
